materialsLibrary/src/water/waterMaterial.ts

```diff
@@ -178,6 +178,12 @@ export class WaterMaterial extends PushMaterial {
     */
     @serialize()
     public waveSpeed: number = 1.0;
+
+    /**
+     * @param {number}: Defines the number of times waves are repeated. This is typically used to adjust waves count according to the ground's size where the material is applied on.
```

Comment:
you don't need the type or even the param keyword for a field

Reply:
Ahah yes I have hesitated, I just followed the way it was commented before. Updating comments right now :)